cluster/addons/podsecuritypolicies-rbac/gce/fluentd-gcp-binding.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: system:podsecuritypolicy:fluentd-gcp
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: system:podsecuritypolicy:fluentd-gcp
+subjects:
+- kind: ServiceAccount
+  # TODO: fluentd-gcp should run with a custom service account.
```

Comment:
I prefer we not start by granting elevated permissions to the default service account... is it small to declare the custom service account?

Reply:
I hear you, but I'm trying to avoid any side affects to this PR. Since this is limited to the kube-system namespace, can I address this in an (immediate) follow up?

Reply:
is it just this?
```diff
diff --git a/cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml b/cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml
index c9cc6bd558..8a0d01df94 100644
--- a/cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml
+++ b/cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml
@@ -1,3 +1,13 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluentd-gcp
+  namespace: kube-system
+  labels:
+    k8s-app: fluentd-gcp
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+---
 apiVersion: extensions/v1beta1
 kind: DaemonSet
 metadata:
@@ -25,6 +35,7 @@ spec:
     spec:
       dnsPolicy: Default
       hostNetwork: true
+      serviceAccountName: fluentd-gcp
       containers:
       - name: fluentd-gcp
         image: gcr.io/google-containers/fluentd-gcp:2.0.9
```

if so, can we open that PR separately and get it merged in parallel with this? seems like it could be in immediately

Reply:
https://github.com/kubernetes/kubernetes/pull/54175